packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -182,6 +182,7 @@ export type ReactNativeType = {
     element: Element<ElementType>,
     containerTag: number,
     callback: ?() => void,
+    concurrentRoot: ?boolean,
```

Comment:
This param isn't on the Paper `render`, only Fabric.

Reply:
Got it, I added it because Flow was still complaining but I found why.
Fabric is using Paper's type (https://github.com/facebook/react/blob/master/scripts/rollup/shims/react-native/ReactFabric.js#L16) 

I changed it in https://github.com/facebook/react/pull/21648/commits/3744c129ff6c90c4592b30f96bb867a1b87c53af

I hope it was fine to remove the todo (cc @sebmarkbage).

Reply:
Nice find.